apps/web/src/pages/templates/workflow/digest/WeekDaySelect.tsx

```diff
@@ -0,0 +1,99 @@
+import { Group, UnstyledButton } from '@mantine/core';
+import { DaysEnum } from '@novu/shared';
+import { useEffect, useState } from 'react';
```

Comment:
please update the imports to what we agreed some time ago: first library imports, then our monorepo, then relative

Reply:
a great catch we should have a lint for this...